test/integration/three_legged_cat_test.go

```diff
@@ -23,7 +23,7 @@ func TestThreeLeggedCat1KBInstantaneous(t *testing.T) {
 		RoutingLatency:    0,
 		BlockstoreLatency: 0,
 	}
-	if err := RunThreeLeggedCat(RandomBytes(1*unit.KB), conf); err != nil {
+	if err := RunThreeLeggedCat(RandomBytes(100*unit.MB), conf); err != nil {
```

Comment:
i think tis is an error. this is supposed to be `TestThreeLeggedCat1KBInstantaneous` ?

I notice that all the rest are 1KB (even the one that says its 100MB): https://github.com/jbenet/go-ipfs/blob/refactors/prep-for-gcr/test/integration/three_legged_cat_test.go#L58